docs/platforms/javascript/common/tracing/span-metrics/examples.mdx

```diff
@@ -1,310 +1,818 @@
+          res.status(400).json({ error: 'Cart is empty' })
+          return
+        }
+
+        let totalMinor = 0
+        for (const line of items) {
+          const product = PRODUCTS.find((p) => p.id === line.productId)
+          if (!product || line.quantity <= 0) {
+            span.setAttribute('payment.status', 'failed')
+            span.setAttribute('inventory.reserved', false)
+            res.status(400).json({ error: 'Invalid cart item' })
+            return
+          }
+          totalMinor += product.priceMinor * line.quantity
+        }
+
+        // Simulate reserving inventory (80% chance true)
+        const reserved = Math.random() < 0.8
+
+        // Simulate payment
+        const charge = await Sentry.startSpan(
+          {
+            name: 'Charge Payment Provider',
+            op: 'commerce.payment',
+            attributes: {
+              'payment.provider': requestedProvider,
+            },
+          },
+          async (paymentSpan) => {
+            const result = await fakeCharge(totalMinor, requestedProvider)
+            paymentSpan.setAttribute('payment.status', result.status)
+            return result
+          }
+        )
+
+        if (charge.status === 'failed' || !reserved) {
+          span.setAttribute('payment.provider', charge.provider)
+          span.setAttribute('payment.status', 'failed')
+          span.setAttribute('inventory.reserved', reserved)
+          res.status(402).json({ error: 'Payment failed' })
+          return
+        }
+
+        const orderId = randomId()
+        ORDERS.push({ id: orderId, totalMinor, items })
+
+        // Set attributes before returning
+        span.setAttribute('order.id', orderId)
+        span.setAttribute('payment.provider', charge.provider)
+        span.setAttribute('payment.status', 'success')
+        span.setAttribute('inventory.reserved', reserved)
+
+        res.json({ orderId, paymentProvider: charge.provider })
+      } catch (err) {
+        Sentry.captureException(err)
+        res.status(500).json({ error: 'Internal error' })
+      }
+    }
+  )
+})
 ```
 
-**How the Trace Works Together:**
-The frontend span initiates the trace and handles the file upload process. It propagates the trace context to the backend through the upload request headers. The backend span continues the trace, processing the file and storing it. This creates a complete picture of the file's journey from client to CDN, allowing you to:
+**How the trace works together:**
+- UI span starts when checkout is selected -> Server Backend starts a span to continue the track when the server `/checkout` API is called. As payment processes, a payment span is started.
+- Attributes and Span metrics let you track more than just the latency of the request. Can track store busienss performances through `cart.item_count` and other `cart` attributes, and store reliabiliyt by checking error performance on `payment.provider` properties.
```

Comment:
There's a typo in here. Should it be "Can track business performance through..."?

Reply:
Also `reliabiliyt`